app/models/post.rb

```diff
@@ -7,6 +7,11 @@ def recent_comments
     comments.order(created_at: :desc).limit(6)
   end
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comment_count, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
```

Comment:
- [ ] Kindly write validation for the attribute `:like_count`. The `:like_count` must be an integer greater than or equal to zero. 👍